tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -67,7 +67,9 @@ def test_slurm(
     Grouped all tests in a single function so that cluster can be reused for all of them.
     """
     scaledown_idletime = 3
-    gpu_instance_type = "g3.4xlarge"
+    # gpu_instance_type = "g3.4xlarge"
+    # TODO Revert to x86_64 gpu_instance_type
+    gpu_instance_type = "g5g.2xlarge"
```

Comment:
you can create a check to use the right instance type according to the architecture of the main instance type configured in the test.
```python
gpu_instance_type = "g3.4xlarge" if instance_architecture_is_arm else "g5g.2xlarge"
```

In this way you don't need to modify the test by hand.

Reply:
As discussed, these are not testing the actual instance that we want. So have removed the commit